README.md

```diff
@@ -2,64 +2,107 @@
-   scheduling of configuration data to remote Kubernetes clusters.
-2. **Providers**: the set of Kubernetes CRDs and controllers that provision and
-   manage services on cloud providers. A cloud provider is any service that
-   exposes infrastructure via an API.
-    - Examples: [Google Cloud
-      Platform](https://github.com/crossplane/provider-gcp), [Amazon Web
-      Services](https://github.com/crossplane/provider-aws),
-      [Azure](https://github.com/crossplane/provider-azure),
-      [Alibaba](https://github.com/crossplane/provider-alibaba),
-      [Github](https://github.com/crossplane/provider-github)
-3. **Stacks**: a bundled set of custom resources that together represent an
-   environment on a cloud provider. The bundle of instances can be created by a
-   single custom resource.
-   - Examples: [GCP Sample
-     Stack](https://github.com/crossplane/stack-gcp-sample), [AWS Sample
-     Stack](https://github.com/crossplane/stack-aws-sample), [Azure Sample
-     Stack](https://github.com/crossplane/stack-azure-sample)
-4. **Applications**: a deployable unit of code and configuration, which, when
-   created, may involve provisioning new services which are managed by a
-   `provider`, or consuming services created by a `stack`.
-    - Examples: [Wordpress](https://github.com/crossplane/app-wordpress)
-
-The full vision and architecture of the Crossplane project is described in our
-[architecture document].
-
-For more information, take a look at the official Crossplane [documentation].
+Crossplane is an open source Kubernetes add-on that extends any cluster with
+the ability to provision and manage cloud infrastructure, services, and
+applications using kubectl, GitOps, or any tool that works with the Kubernetes
+API.
+
+With Crossplane you can:
+
+* **Provision & manage cloud infrastructure and services with kubectl**
+  * [Install Crossplane] to provision and manage cloud infrastructure and
+    services from any Kubernetes cluster.
+  * Provision infrastructure primitives from any provider (GCP, AWS, Azure,
+    Alibaba, on-prem) and use them alongside existing application configurations.
+  * Version, manage, and deploy with your favorite tools and existing
+    workflows that you’re using with your clusters today.
+
+* **Publish custom infrastructure resources for your applications to use**
+  * Define, compose, and publish your own [infrastructure resources] with
+    declarative YAML, resulting in your own infrastructure CRDs being added to
+    the Kubernetes API for applications to use.
+  * Hide infrastructure complexity and include policy guardrails, so
+    applications can easily and safely consume the infrastructure they need,
+    using any tool that works with the Kubernetes API.
+  * Consume infrastructure resources alongside any Kubernetes application to
+    provision and manage the cloud services they need with Crossplane as an
+    add-on to any Kubernetes cluster.
```

Comment:
I notice "with Crossplane as an add-on to any Kubernetes cluster" repeats the first sentence in this document. Is it useful here? It seems like it's implied for every item in this list.

Reply:
I was just trying to drive that point home with repetition, but happy to remove it if it's too annoying. 🙂